src/main/java/com/mraof/minestuck/entity/item/GristEntity.java

```diff
@@ -96,11 +118,73 @@ public boolean hurt(DamageSource source, float amount)
 		}
 	}
 	
+	public void setAnimationFromPacket(Animation animation)
+	{
+		switch(animation)
+		{
+			case REJECT:
+				shaderAlpha = 255;
+				break;
+		}
+	}
+	
+	public enum Animation
+	{
+		REJECT
+	}
```

Comment:
What's the purpose of this with the only value being `REJECT`?

Reply:
wasn't sure where else to put it

Reply:
There's not much point to it existing if its only possible value is REJECT.